Update az cli args

noelbundick opened the pull request:
Updated arguments to fix error encountered in Azure Cloud Shell

Versions:
azure-cli 2.0.14
container component 0.1.8

Thraka commented:
Two of the articles do not have an official author. This is known and to be addressed in the future by the manager.

noelbundick committed:
Update az cli args

Updated arguments to fix error encountered in Azure Cloud Shell

Versions:
azure-cli 2.0.14
container component 0.1.8
